src/Application/ApplicationFileProcessor.php

```diff
@@ -58,14 +58,15 @@ public function __construct(
     public function run(Configuration $configuration, InputInterface $input): ProcessResult
     {
         $filePaths = $this->filesFinder->findFilesInPaths($configuration->getPaths(), $configuration);
-        $this->missConfigurationReporter->reportVendorInPaths($filePaths);
-        $this->missConfigurationReporter->reportStartWithShortOpenTag();
 
         // no files found
         if ($filePaths === []) {
             return new ProcessResult([], []);
         }
 
+        $this->missConfigurationReporter->reportVendorInPaths($filePaths);
+        $this->missConfigurationReporter->reportStartWithShortOpenTag();
```

Comment:
This will not report the notice on 2nd run if no change, as list of file paths cached, which make confusing if user run twice, the report notice is gone.

Reply:
oh I see, if no filepaths, then nothing to check, probably original `$configuration->getPaths()` passed is needed, but that's different story to discuss then.

It is ok as is for this change 👍